html-css/week1/bayram/order-desktop/styles.css

```
@@ -0,0 +1,200 @@
+  font-weight: bold;
+  font-size: 24px;
+  text-align: left;
+  color: white;
+  margin-bottom: 15px;
+}
+.total-container {
+  color: white;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+.total-bold {
+  font-weight: bold;
+}
+.separator {
+  background-color: rgba(255, 255, 255, 0.5);
+  margin: 5px 0;
+  width: 100%;
+  height: 1.5px;
+}
+
+.payment-title {
+  margin: 0;
+  color: black;
+}
+
+.payment-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: flex-start;
+  gap: 20px;
+  padding: 20px;
+}
+
+.box-container {
+  display: flex;
+  gap: 20px;
+  justify-content: center;
+  width: 100%;
+}
+
+.payment-box {
+  padding: 10px;
+  text-align: center;
+  width: 50%;
+  height: auto;
+  background-size: contain;
+  background-repeat: no-repeat;
+  background-position: center;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  border: 2px solid transparent;
+  border-radius: 12px;
+  background-color: #fff;
+  box-shadow: 0 18px 14px rgba(0, 0, 0, 0.1);
+  cursor: pointer;
+  transition: all 0.3s ease-in-out;
+}
+
+.payment-box:hover {
+  border: 3px solid #ff7eb3;
+  transform: scale(1.05);
+}
+
+.payment-image {
+  width: 60%;
+  height: 50px;
+  object-fit: contain;
+  text-align: center;
+}
+
+.payment-text {
+  font-weight: 600;
+  font-size: 18px;
+  color: #939393;
+}
+
+.card-container {
+  margin: 20px auto;
+  padding: 20px;
+  border-radius: 12px;
+}
+
+.card-input-container {
+  display: flex;
+  flex-direction: column;
+  position: relative;
+}
+.card-number {
+  width: 342px;
+}
+
+.card-expiry {
+  width: 162px;
+}
+
+.card-security {
+  width: 199px;
+}
+
+.card-input-container label {
+  margin-bottom: 5px;
+  color: #999;
+  text-align: center;
+  font-weight: 600;
+  font-size: 18px;
+}
+
+.card-input-container input {
+  padding: 10px;
+  border: 1px solid #ccc;
+  border-radius: 10px;
+  font-size: 1rem;
+  background-color: #f3f1f2;
+}
+
+.tooltip {
+  position: absolute;
+  top: 40px;
+  right: 10px;
+  background-color: #666;
+  color: #fff;
+  border-radius: 5px;
+  font-size: 0.9rem;
+  border-radius: 50%;
+  width: 20px;
+  height: 20px;
+  text-align: center;
+  line-height: 20px;
+  cursor: pointer;
+}
+
+.multi-card-container {
+  display: flex;
+  gap: 20px;
+  flex-wrap: wrap;
+  justify-content: center;
+}
+
+.card-info {
+  margin-bottom: 20px;
+  text-align: center;
+  color: black;
+}
+
+.buy-now {
+  background: linear-gradient(to right, #fd7b46, #fe4e9b);
+  width: 326px;
+  color: white;
+  font-size: 1.5rem;
+  font-weight: bold;
+  padding: 15px;
+  border: none;
+  border-radius: 30px;
+  cursor: pointer;
+  margin: 20px auto;
+  box-shadow: 0 6px 8px rgba(0, 0, 0, 0.1);
+  transition: all 0.3s ease-in-out;
+  display: block;
+}
+
+.buy-now:hover {
+  background: linear-gradient(to left, #ff758c, #ff7eb3);
+  box-shadow: 0 6px 8px rgba(0, 0, 0, 0.2);
+}
```

Nice effect. If you add `:focus-visible` to the selector, then you would apply the same effect when someone reaches the button through using the tab key.
```suggestion
.buy-now:hover,
.buy-now:focus-visible {
  background: linear-gradient(to left, #ff758c, #ff7eb3);
  box-shadow: 0 6px 8px rgba(0, 0, 0, 0.2);
}
```